### PR TITLE
Tracking Updates

### PR DESCRIPTION
**Pull request by wtgee**
* Wait for the Dec axis to move before moving RA axis
* Only update tracking if > 50ms correction
* Some long line clean up

Note: Most of this `update_tracking` code should move into the mount
class and accept `current_offset_info` as a param

Corrects for small adjustments as reported in #351 
Relates to #350

### CODE REVIEW

File: pocs/observatory.py [hunk 1]
```diff
@@ -148,12 +149,15 @@ def status(self):
                 status['observation']['field_ha'] = self.observer.target_hour_angle(
                     t, self.current_observation.field)
 
+            evening_astro_time = self.observer.twilight_evening_astronomical(t, which='next')
```
Comment (jamessynge) [inline pocs/observatory.py]: What does next produce if at midnight? Tomorrow night?

File: pocs/observatory.py [hunk 2]
```diff
@@ -316,36 +320,48 @@ def update_tracking(self):
             else:
                 ra_direction = 'east'
```
Comment (jamessynge) [inline pocs/observatory.py]: Could you add some more explanation to the docstring, for those of us who are ignorant of mounts. For example, what does 50 milliseconds measure here? An angle? A time?

File: pocs/observatory.py (hunk 1 above)
Comment (wtgee) [reply pocs/observatory.py]: Correct. The default is `nearest`, which would go backwards. With `next` we always get the upcoming.

File: pocs/observatory.py (hunk 2 above)
Comment (wtgee) [reply pocs/observatory.py]: Yes, will add.

Comment (wtgee) [reply pocs/observatory.py]: Done. I'm going to merge but let me know if docstring doesn't make sense. I'd like to test these additions on PAN001 tonight.

**Pull request merged**